lesson05/shape_test.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestDescribeShape(t *testing.T) {
+	r0 := Rectangle{
+		Height: 0,
+		Width:  0,
+	}
+	r5 := Rectangle{
+		Height: 5,
+		Width:  5,
+	}
+	c0 := Circle{Radius: 0}
+	c1 := Circle{Radius: 1}
+
+	negativeCircleResult := DescribeShape(c0)
```

Comment:
You can't do this, `DescribeShape` doesn't return anything. Generally speaking this function should not be tested as it doesn't have return statements

Reply:
shape_test.go deleted